websites/C/Chiasenhac/dist/metadata.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.6",
+  "author": {
+    "name": "Kyrie",
+    "id": "368399721494216706"
+  },
+  "service": "Chiasenhac",
```

Comment:
is that an official version of the name? Otherwise just use the version in altnames

Reply:
tbh I would still put the original/primary name and then the other as altname. I am guessing the 2 should be switched around.

Reply:
Will change ASAP